.env-dist

```diff
@@ -100,4 +100,7 @@ EDUCATION_VIDEO_URL_VPN=https://monitor.cdn.mozilla.net/videos/Mozilla_VPN.mp4
 ADMINS=
 
 # Enable monthly cron-job, currently for sending unresolved breach reminder emails
-MONTHLY_CRON_ENABLED=
\ No newline at end of file
+MONTHLY_CRON_ENABLED=
+
+SERVER_GENERATED_NONCE=test123
```

Comment:
NOTE - this is just for testing, we need to generate this on every request. I'll fix this before moving this PR out of draft mode.